### PR TITLE
Dev me20 examples

### PR DESCRIPTION
**Pull request by ozersa**
MAX32680 examples reviewed.
Detail [here](https://confluence.maxim-ic.com/display/MSW/MSDK+Example+Validation+and+Adding+Missing+Boards)
Unresolved JIRA tickets
[MAX32680](https://jira.maxim-ic.com/browse/MSDK-658)

### CODE REVIEW

**Review by Jake-Carter — CHANGES_REQUESTED** [review]
Looks good.  Only thing to resolve is the return value mismatch

File: Examples/MAX32680/AES/main.c [hunk 1]
```diff
@@ -145,15 +144,11 @@ int main(void)
...
         printf("\nExample Succeeded\n");
     } else {
-        printf("Example Failed\n");
+        printf("\nExample Failed\n");
     }
 
-    while (1) {}
-
-    return 0;
+    return fail;
```
[the remark this answers is not in the document]
Comment (ozersa) [reply Examples/MAX32680/AES/main.c]: In fact "0" is meaning value, for fail cases any non-zero values should be fine.
Anyway reverted.

File: Examples/MAX32680/DMA/main.c [hunk 2]
```diff
@@ -225,17 +224,11 @@ int main(void)
...
-        printf("Example Failed\n");
-
-        while (1) {}
+        printf("\nExample Failed\n");
     }
 
-    while (1) {}
-
-    return 0;
+    return fail;
```
[the remark this answers is not in the document]
Comment (ozersa) [reply Examples/MAX32680/DMA/main.c]: Thanks I was aware of it.
To simplify it reverted some changes.

File: Examples/MAX32680/LP/main.c [hunk 3]
```diff
@@ -164,32 +170,25 @@ void setTrigger(int waitForTrigger)
 
 int main(void)
 {
-#if USE_CONSOLE
-    printf("****Low Power Mode Example****\n\n");
-#endif // USE_CONSOLE
+	PRINT("****Low Power Mode Example****\n\n");
```
[the remark this answers is not in the document]
Comment (ozersa) [reply Examples/MAX32680/LP/main.c]: Thanks, done.

**Review by Jake-Carter — APPROVED** [review]

File: Examples/MAX32680/AES/main.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jake-Carter) [reply Examples/MAX32680/AES/main.c]: Thanks.  I think it's just good to have consistency.  It may save us some headaches in the future

**Comment by sihyung-maxim** [comment]
I've been holding on approving some of the PRs until the register updates are pushed in.

**Review by lorne-maxim — APPROVED** [review]

**Review by sihyung-maxim — APPROVED** [review]

**Pull request merged**